app.py

```diff
@@ -1,1268 +1,174 @@
 import flask_login
-import git
-from git import Repo
-from github import Github
-import json
+import logging
 import os
 import pyrebase
-import requests
-import slack
-from slackeventsapi import SlackEventAdapter
-import shutil
-import shlex
-import time
 import threading
-from urllib.parse import parse_qs
-from werkzeug.utils import secure_filename
-import yaml
-from yaml import load, dump
 
-tars_token = os.environ.get("TARS_TOKEN")
-tars_user_token = os.environ.get("TARS_USER_TOKEN")
-tars_admin = os.environ.get("TARS_ADMIN")
-tars_secret = os.environ.get("TARS_SECRET")
-tars_bot_id = os.environ.get("TARS_BOT_ID")
-tars_id = os.environ.get("TARS_ID")
-general_id = os.environ.get("GENERAL_ID")
-orientation_id = os.environ.get("ORIENTATION_ID")
-project_id = os.environ.get("PROJECT_ID")
-sf_research = os.environ.get("SF_RESEARCH")
-sf_ta = os.environ.get("SF_TA")
-vineethv_id = os.environ.get("VINEETHV_ID")
-office_hours_form = os.environ.get("OFFICE_HOURS_FORM")
-firebase_api_key = os.environ.get("FIREBASE_API_KEY")
-tars_fb_ad = os.environ.get("TARS_FB_AD")
-tars_fb_url = os.environ.get("TARS_FB_URL")
-tars_fb_sb = os.environ.get("TARS_FB_SB")
-key_fb_tars = os.environ.get("KEY_FB_TARS")
-hyouka_fb_key = os.environ.get("HYOUKA_FB_KEY")
-hyouka_fb_ad = os.environ.get("HYOUKA_FB_AD")
-hyouka_fb_url = os.environ.get("HYOUKA_FB_URL")
-hyouka_fb_sb = os.environ.get("HYOUKA_FB_SB")
-key_fb_hyouka = os.environ.get("KEY_FB_HYOUKA")
-github_secret = os.environ.get("GITHUB_SECRET")
-username = os.environ.get("USERNAME")
-password = os.environ.get("PASSWORD").encode()
-github_username = os.environ.get("GITHUB_USERNAME")
-github_email = os.environ.get("GITHUB_EMAIL")
-github_password = os.environ.get("GITHUB_ACCESS_TOKEN")
-secret = os.environ.get("SECRET")
+from dotenv import load_dotenv
+from helpers import *
+from slack_bolt import App
+from slack_bolt.adapter.flask import SlackRequestHandler
+from flask import Flask, jsonify, redirect, request, render_template, redirect, url_for
 
-app = Flask(__name__)
-app.secret_key = secret
-app.debug = True
+load_dotenv()
 
-login_manager = flask_login.LoginManager()
-login_manager.init_app(app)
-login_manager.login_view = "login"
 
-class User(flask_login.UserMixin):
-    pass
+tars_token = os.environ.get("TARS_TOKEN")  # bot user OAuth
+tars_user_token = os.environ.get("TARS_USER_TOKEN")  # user OAuth
+tars_admin = os.environ.get("TARS_ADMIN")  # channel for testing
+tars_secret = os.environ.get("TARS_SECRET")  # signing secret
 
-@login_manager.user_loader
-def load_user(id):
-    if id != username:
-        return
-    user = User()
-    user.id = id
-    return user
+firebase_api_key = os.environ.get("FIREBASE_API_KEY")
+tars_fb_ad = os.environ.get("TARS_FB_AD")  # firebase auth domain
+tars_fb_url = os.environ.get("TARS_FB_URL")  # database base url
+tars_fb_sb = os.environ.get("TARS_FB_SB")  # storage bucket
+key_fb_tars = os.environ.get("KEY_FB_TARS")  # the long sha
```

Comment:
Let's align the comment strings (and make them neater) if we're going to keep them.

Reply:
black is autoformatting to this form :3